build.gradle

```diff
@@ -17,5 +17,5 @@ android {
 }
 
 dependencies {
-  implementation 'com.google.android.gms:play-services-ads-lite:22.2.0'
+  implementation "com.google.android.gms:play-services-ads-identifier:$googlePlayServicesAdIdVersion"
```

Comment:
is `googlePlayServicesAdIdVersion` defined in the flagship repo? if so, we probably should not reference it inside of a submodule, since in theory this repo should be able to be cloned and build on its own